decidim-core/app/validators/in_organization.rb

```diff
@@ -0,0 +1,17 @@
+module Decidim
+  class FeatureValidator < ActiveModel::EachValidator
```

Comment:
Or maybe it should be removed, since the code is doing the same as `FeatureValidator` and it's not being used